README.md

```diff
@@ -188,6 +184,21 @@ It is hard knowing what is going on inside of some of these ADTs or your wonderf
 #### `tryCatch : (a -> b) -> a -> Either e b`
 Typical try-catch blocks are very imperative in their usage. This `tryCatch` function provides a means of capturing that imperative nature in a simple declarative style. Pass it a function that could fail and it will return you another function wrapping the first function. When called the new function will either return the result in an `Either.Right` if everything was good, or an error wrapped in an `Either.Left` if it fails.
 
+### Logic Functions
+The functions in this section are used to represent simple to complex logical branching in a declarative manner. All of these function will take predicate functions or `Pred`s and either return a resulting `Boolean` or run some function based on the result of the predicates. All of these functions work with functions/`Pred`s instead of values to allow for building out complex, "lazy" evaluated and allow easy composition.
+
+#### `and : ((a -> Boolean) | Pred) -> ((a -> Boolean) | Pred) -> a -> Boolean`
+Say you have two predicate functions or `Pred`s and would like to combine them into one predicate over conjunction, well you came to the right place, `and` accepts either predicate functions or `Pred`s and will return you a function ready to take a value. Once that value is passed, it will run it through both of the predicates and return the result of combining it over a `logical and`. This is super helpful combined with `or` for putting together reusable, complex predicates. As they follow the general form of `(a -> Boolean)` they are easily combined with other logic functions.
```

Comment:
The Majority of your descriptions read as if you REALLY want to get your word across, but you don't contextualize 🥂  [in essence, instead of trying to sound human with "Say you have two predicate functions", you can just sound like a robot here with "Combinatorial mathematics"